File: FibonacciSeq/fibonacci.cpp
#include<iostream>
using namespace std;

int fib(int n);

int main()
{
    int n;

    cout << "Enter a positive integer: ";
    cin >> n;
	cout << "Fib of " << n << " = " << fib(n) << std::endl;
    return 0;
}

int fib(int n)
{
	if(n == 0 || n == 1)
	{
		return 1;
	}
    else if(n > 1)
    {
        return (fib(n - 2) + fib(n - 1));
    }
}

File: FibonacciSeq/Fibonacci.rb
def fib(n)
    if n <= 2
        return 1
    else
        return fib(n-1) + fib(n-2)
    end
end

puts "Enter limit : "
limit = gets.to_i

result = fib(limit)

puts "Fib(#{limit}) = #{result}"